File: 1979-find-greatest-common-divisor-of-array/1979-find-greatest-common-divisor-of-array.py
class Solution:
    def s(self,a,b):
        if(b==0):
            return a
        return self.s(b,a%b)
    def findGCD(self, nums: List[int]) -> int:
        nums.sort()
        a=nums[0]
        b=nums[-1]
        return self.s(a,b);

File: 1979-find-greatest-common-divisor-of-array/1979-find-greatest-common-divisor-of-array.cpp
class Solution {
public:
    int send(int a,int b)
    {
        if(b==0)
            return a;
        return send(b,a%b);
    }
    int findGCD(vector<int>& nums) {
        sort(nums.begin(),nums.end());
        int a=nums[0];
        int b=nums[nums.size()-1];
      return send(a,b);
    }
};